src/webapi/events/mouse.rs

```diff
@@ -439,6 +439,87 @@ impl ConcreteEvent for MouseLeaveEvent {
     const EVENT_TYPE: &'static str = "mouseleave";
 }
 
+/// The `WheelEvent` is fired when a pointing device's wheel button (usually a mousewheel)
+/// is rotated over the element that has the listener attached.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/Events/mousewheel)
+// https://w3c.github.io/uievents/#event-type-wheel
+#[derive(Clone, Debug, PartialEq, Eq, ReferenceType)]
+#[reference(instance_of = "MouseEvent")] // TODO: Better type check.
+#[reference(subclass_of(Event, UiEvent, MouseEvent))]
+pub struct MouseWheelEvent( Reference );
+
+impl IEvent for MouseWheelEvent {}
+impl IUiEvent for MouseWheelEvent {}
+impl IMouseEvent for MouseWheelEvent {}
+impl ConcreteEvent for MouseWheelEvent {
+    const EVENT_TYPE: &'static str = "mouseleave";
```

Comment:
Isn't thins wrong? (:

Reply:
Whoops, the dangers of copy-paste I guess! Should be fixed with latest commit.